### DISCUSSION TITLE
Nova Modal - ActionModal

### DISCUSSION THREAD

**Discussion by RealityGamin** [laravel/nova-issues#4417] [Q&A]
I'm really confused on what components are available to use and what components are not.

I tried in Tools.vue
```
<template>
	<div>
		<Modal show="true" />
		<ConfirmActionModal show="true" />
	</div>
</template>
```

But ConfirmActionModal does not work, I found few guides on how to use modals with laravel nova, but they are 3+ years old, could someone please share with me just a basic skeleton to use nova modals? even better if i can get ConfirmActionModal to work.

I'm simply trying to make a button in my Tool.vue page where a confirmation message will appear and you can click confirm/cancel. I don't have the slightest idea how to accomplish this with Nova Modal

I appreciate it thank you.

**Comment by chrisbjr** [comment]
it's 2024 and i still don't see a way to use Nova's modals inside custom components
